requirements/base.txt

```diff
@@ -1,3 +1,23 @@
-django>=1.8,<2.0
+# edX Internal Requirements
+# edx-submissions>=2.0.6,<3.0.0 #TODO: update after 2.0.6 release
+git+https://github.com/edx/edx-submissions.git@efischer/loosen_reqs#egg=edx-submissions==2.0.6 # TODO: and remove this line
+git+https://github.com/edx/django-rest-framework.git@1ceda7c086fddffd1c440cc86856441bbf0bd9cb#egg=djangorestframework==3.6.3
```

Comment:
@doctoryes set this requirement to a specific commit in https://github.com/edx/edx-ora2/pull/1024, I'm just moving it to another file